src/components/CreateProfileBtn/CreateProfileBtn.tsx

```diff
@@ -1,17 +1,23 @@
 'use client'
 import React from 'react'
 import { Button } from '@/components/Button/Button'
-import { AppRoutes } from '@/utils/routes'
-import { useRouter } from 'next/navigation'
 import AddIcon from '@/assets/icons/AddIcon'
+import { signIn, useSession } from 'next-auth/react'
+import { AppRoutes } from '@/utils/routes'
 
 const CreateProfileBtn = () => {
-  const router = useRouter()
+  const { data: session } = useSession()
+
+  const onClickHandler = async () => {
+    if (!session) {
+      await signIn('github', {
+        callbackUrl: AppRoutes.githubOAuth,
+      })
```

Comment:
**issue:** user can be logged in without profile. we should `router.push` to create profile page, when user is logged in but does not have profile